sdk/lib/3rdparty/freetype/src/autofit/aflatin2.c

```diff
@@ -80,7 +84,11 @@
       FT_Error             error;
       FT_UInt              glyph_index;
       int                  dim;
+#ifdef __REACTOS__
+      AF_LatinMetricsRec  *dummy = calloc(1, sizeof(*dummy));
```

Comment:
NULL check missing (see the other function as well): `if (!dummy) goto Exit;`
(and an opening bracket for C89 compatibility)

Reply:
Added.